blueprint-manager/src/main/java/com/sequenceiq/cloudbreak/blueprint/GeneralClusterConfigsProvider.java

```diff
@@ -65,7 +66,7 @@ public GeneralClusterConfigs generalClusterConfigs(StackV2Request stack, Identit
         }
 
         GeneralClusterConfigs generalClusterConfigs = new GeneralClusterConfigs();
-        generalClusterConfigs.setIdentityUserEmail(identityUser.getUsername());
+        generalClusterConfigs.setIdentityUserEmail(user.getUserId());
```

Comment:
I see your concern here, it looks a little dangerous. When we will have proper email field for the user, it can be changed to that. We will see how it evolves with the new authentication method later.